logos/tests/langchainTest.py

```
@@ -0,0 +1,107 @@
+        headers = {
+            "Content-Type": "application/json",
+            "Authorization": f"Bearer {self.logos_key}",
+            "deployment_name": self.deployment_name,
+            "api_version": self.api_version
+        }
+        prompt_parts = []
+        for msg in messages.messages:
+            if isinstance(msg, HumanMessage):
+                prompt_parts.append(msg.content)
+            elif isinstance(msg, SystemMessage):
+                prompt_parts.append(msg.content)
```

Ruff (0.11.9)

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)